packages/fast-components-react-msft/src/button/button.props.ts

```diff
@@ -2,26 +2,18 @@ import * as React from "react";
 import { IButtonHandledProps as IBaseButtonHandledProps } from "@microsoft/fast-components-react-base";
 import { IManagedClasses, IMSFTButtonClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
 
-export interface IButtonHandledProps extends IBaseButtonHandledProps {
-    /**
-     * The lightweight justified option
-     */
-    justified?: boolean;
-
-    /**
-     * The lightweight option
-     */
-    lightweight?: boolean;
-
-    /**
-     * The outline option
-     */
-    outline?: boolean;
+export enum Appearance {
```

Comment:
I think we should namespace this specific to button to ensure we don't create a global that we may need to leverage later. Similar components with similar enum's would follow the same pattern.

Thoughts?

Reply:
The prop name is fine because it is scoped, but here I think we'll want to specify that this is `ButtonAppearance`